### ISSUE TITLE
Informe de Accidente

### ISSUE THREAD

**Issue by Nikelondeon** [proyectoalba/alba2#15]
Si el alumno se accidenta en la Institución (aula, patio de recreo, gimnasio), que se pueda registrar brevemente el episodio, se identifiquen los adultos que acudieron, el directivo que toma conocimiento del episodio y como obró el docente a cargo de la situación (que se consideró, a cual de los padres se les avisó, que familiar acudió a la institución, llegada de la ambulancia, indicación médica, con quien se retiró y horario de retiro del alumno.

**Comment by lucianobaraglia** [comment]
Me gusta!
Podríamos llamarlo **Incidencia**...¿qué te parece?
Porque no sólo sería accidentes, sino podrían registrarse cualquier otra cosa: peleas, desaparición de alguna pertenencia, etc.
Varios alumnos podrían estar relacionados a la misma, por si fueron varios participantes.

**Comment by Nikelondeon** [comment]
Sí, me gusta. Es más, hay una Resolución específica del Ministerio de Educación que dictamina el modus operandi ante el accidente de un alumno. Sería ideal que se copien los formularios.
Después comparto los archivos si te parece bien.

**Comment by lucianobaraglia** [comment]
Claro!!! Toda data oficial nos viene de maravilla!!! :+1:

**Comment by Nikelondeon** [comment]
Para que vayan trabajando si quieren, cuando se produce un Accidente o se lastima el alumno por algún otro motivo, la Escuela debe tener la siguiente documentación producida:

SINTESIS DEL HECHO
FOTOCOPIA DNI ALUMNO
FOTOCOPIA REGISTRO MATRICULA
FOTOCOPIA REGISTRO AULA
ACTA DESCRIPTIVA
ACTA CON FAMILIARES
ACTA OFRECIMIENTO SEGURO ESCOLAR
PLANILLA SEGURO ESCOLAR
DIAGNOSTICO MEDICO (en caso de haber sido atendido por un médico)
DENUNCIA A PROVINCIA SEGUROS
EXPOSICION CIVIL o DENUNCIA POLICIAL
OTROS ACTUADOS (cualquier cosa que tenga que ver con el hecho)
EMISION DE CRITERIO DEL DIRECTIVO

No todo es obligatorio, depende el tipo de Accidente que haya sufrido.
Quizás se pueden trabajar las Actas desde el ALBA, o ante un Accidente que sirva como checks para que no falta producir nada.

**Comment by lucianobaraglia** [comment]
Fijáte que esto creo que está muy atado a las escuelas secundarias de Provincia de Buenos Aires, por lo cual tenemos que intentar hacerlo un poco más genérico, para que sirva a otros niveles de otras jurisdicciones.

Lo que se me ocurre es lo siguiente: podríamos que hacer que cuando sucede una de estas incidencias, se puedan adjuntar archivos, cualquier cantidad, digamos.
Sería subir el/los archivo/s al cargar la incidencia, y poner un título y una descripción del archivo que se está subiendo.

O sea, en la incidencia se poner título, descripción y alumno.
Y ahí subir _n_ archivos, cada uno con título y pequeña descripción.

¿Qué opinás?

**Comment by Nikelondeon** [comment]
Buena idea lo de subir archivos. Me gusta.

**Comment by lucianobaraglia** [comment]
Ver https://github.com/proyectoalba/alba2/issues/16#issuecomment-38887089
